### DISCUSSION TITLE
Internal Viewer - primary selection support (auto-copy on select, paste with mid mouse button)

### DISCUSSION THREAD

**Discussion by zorba128** [doublecmd/doublecmd#1815] [Suggestions]
Would it be possible to add support for linux-style primary selection to internal viewer? 

Or maybe it is somehow broken - I checked and it works fine in built-in editor.
But after selecting something in viewer, primary selection buffer gets always emptied.

marcin

**Comment by alexx2000** [comment]
It is not supported in the viewer. It can be added.
